### PR TITLE
Added install-latest Makefile target to allow testing of ecr image

### PR DESCRIPTION
**Pull request by bmsiegel**
Signed-off-by: Brady Siegel <brsiegel@amazon.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -274,6 +274,15 @@ install-local: docker-build docker-push-local
 	--set serviceAccount.create=false --set serviceAccount.name=${SERVICE_ACCOUNT} \
 	--set image.repository=${LOCAL_IMAGE} --set image.tag=latest --set image.pullPolicy=Always
 
+.PHONY: install-latest
```
Comment (solonish) [inline Makefile:277]: can we change the name to install-test or install-latest-test? as we are installing from the test ECR.

File: Makefile [hunk 2]
```diff
@@ -285,6 +294,8 @@ upgrade-local: uninstall-local install-local
 .PHONY: cluster
 cluster: manager create-local-registry kind-cluster deploy-cert-manager install-local
 
+.PHONY: cluster-latest
```
Comment (solonish) [inline Makefile:297]: can we change the name to cluster-ecr-test or cluster-test?

**Comment by divyansh-gupta** [comment]
How do you feel about replacing the `test` with `beta` throughout? I'd like our test repository to turn into an easy to use beta program one day - plugin beta users would probably like `install-beta-ecr-image` a bit more than `install-beta-test-image`. Small-nitpick.

**Review by divyansh-gupta — APPROVED** [review]

**Pull request merged**